recipes/emsdk_installer/all/conanfile.py

```diff
@@ -0,0 +1,121 @@
+from conans import ConanFile, tools
+import os
+
+
+class EmSDKInstallerConan(ConanFile):
+    name = "emsdk_installer"
+    description = "Emscripten is an Open Source LLVM to JavaScript compiler"
+    url = "https://github.com/bincrafters/conan-emsdk_installer"
+    homepage = "https://github.com/kripken/emscripten"
+    topics = ("conan", "emsdk", "emscripten", "installer", "sdk")
+    license = "MIT"
```

Comment:
It needs access to the `os` on which it runs.
```suggestion
    license = "MIT"
    settings = "os"
```